README.md

```diff
@@ -93,14 +120,22 @@ The following source code exposes the `bar` function to the public API for testi
 
 
 #### Specifying different start and end comment values
-The following configuration will strip out code that begins with the `/* start-test-block */` comment and ends with the `/* end-test-block */` comment from all `.js` files in the `dist/` folder.
+The following configuration will strip out code that begins with the `/* start-test-block */` comment and ends with the `/* end-test-block */` comment, and code that begins with the `<!-- start-html-test-code -->` comment and ends with the `<!-- end-html-test-code -->` comment from all `.js` files in the `dist/` folder.
 
 ```js
 grunt.initConfig({
   strip_code: {
     options: {
-      start_comment: 'start-test-block',
-      end_comment: 'end-test-block',
+      blocks: [
+        {
+          start_block: "/* start-test-block */",
+          end_block: "/* end-test-block */"
+        },
+        {
+          start_block: "<!-- start-html-test-code -->",
+          end_block: "<!-- end-html-test-code -->"
+        }
+      ]
     },
     src: 'dist/*.js'
```

Comment:
@ZuBB in reply to your question below/earlier. In the line here, you can only specify directories or files to strip from, which might end up forcing you to list all but a few files if you wanted to strip using a pattern rather than a start and stop block. For example trying to use a regex pattern to remove all uses of "console.log" in all files of project except for files that need to show logs for reporting or debugging purposes.